scripts/main/lychee_locale.js

```diff
@@ -9,6 +9,7 @@ lychee.locale = {
 	SETTINGS: "Settings",
```

Comment:
I don't remember you updating the locale files on the server side? You'll have to add these strings to every language file (sorry!).

Reply:
Alright, so just put entries in each language file, just in English?

Reply:
In all language files. We have the CI that checks that every lines in the English language is at least found in the other languages file (as place-holders).

**We do not expect you to do the translations**. Just copy over the same line as in the `English.php` file. That way, other users can directly look at their language file and edit the wrong/missing translations rather than having to find the missing lines between the two files.